test/sample_set_filter_test.py

```diff
@@ -220,3 +220,9 @@ def test_order_of_ops(self):
             }
         ])
         self.assertEqual(len(samples_merge), 206)
+
+    def test_get_sampleset_meta(self):
+        ret = self.serviceImpl.get_sampleset_meta({'sample_set_ref': self.sample_set_2_ref})
+        self.assertEqual(type(ret), list)
+        self.assertEqual(type(ret[0]), str)
+        self.assertIn('isgn', ret)
```

Comment:
I am not sure what search API return looks like. Something like `['isgn', 'bar', 'foo'...]`? Can we do assertCountEqual to example the whole return?

Reply:
I tried to mimic the results based off of what I can see from sample_service_validator_config, but I wasn't sure what in the test csv was a custom field or not, so this test may need some tweaking